docker/lwua-ingest/lwua-py/tests/test_queries.py

```diff
@@ -0,0 +1,107 @@
+# test file to test the JRDF builder and all the templates
+
+from pyrdfj2 import J2RDFSyntaxBuilder
+from lwua.helpers import resolve_path
+import os
+
+URN_BASE = os.getenv("URN_BASE", "urn:lwua:INGEST")
+
+
+def get_j2rdf_builder():
+    template_folder = os.path.join(
+        os.path.dirname(__file__),
+        "../lwua/templates")
+    # init J2RDFSyntaxBuilder
+    context = f"{URN_BASE}:ADMIN"
+    j2rdf = J2RDFSyntaxBuilder(
```

Comment:
not doable to just reuse the syntax-builder in graph.py?

if so you would be testing that too

Reply:
Not really, the template folder location is different